modules/indigo/module/inc/indigo/of_state_manager.h

```diff
@@ -329,6 +407,24 @@ indigo_core_gentable_register(
 void
 indigo_core_gentable_unregister(indigo_core_gentable_t *gentable);
 
+/*
+ * @brief resume indigo addi4()/modifyi4()/del4() when the driver gets the final status 
+ * @param op_ctx operation context 
+ * @param entry_priv driver's private data for this entry
+ * @param err_txt error string from driver
+ * @param rv the final status of the driver async add/modify
```

Comment:
i think the resume function will apply to add, modify, and delete; please confirm and correct

Reply:
correct. Fixed